static/js/settings_account.js

```diff
@@ -410,7 +408,7 @@ exports.set_up = function () {
             error_msg_element: change_password_error,
         };
         settings_ui.do_settings_change(channel.patch, '/json/settings', data,
-                                       $('#account-settings-status').expectOne(), opts);
+                                       $('.email-change-form').expectOne(), opts);
```

Comment:
You are supposed to `status_element` as a 4th argument here?

Reply:
No. I have changed the do_settings_change function. The status element is a element which contains the settings_save_discard widget. Couldn't find a proper #id selector so had to use a class, should I assign an id to the div to keep the code uniform?